packages/paste-website/src/pages/components/meter/index.mdx

```diff
@@ -0,0 +1,235 @@
+
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {getFeature, getNavigationData} from '../../../utils/api';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {defaultMeter, minMaxMeter, hiddenValueLabel} from '../../../component-examples/MeterExamples';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Meter');
+  return {
+    props: {
+      data: {
+        ...packageJson,
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  storybookUrl="/?path=/story/components-meter--"
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/meter"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+<LivePreview scope={{Meter, MeterLabel, HelpText, useUID, Box}} language="jsx" noInline>
+  {defaultMeter}
+</LivePreview>
+
+## Guidelines
+
+### About Meter
+
+A Meter is a visual representation to indicate how full something is.
+
+### Meter vs. Progress Bar
+
+A Meter represents a bucket that can be empty, full, or somewhere in between. Use a Meter when you need to show capacity. For example, use a Meter to show how much data is being used or how many emails were sent successfully.
+
+A [Progress Bar](/components/progress-bar) represents **only** task completion, like a file upload or filling out a form. If you’re not displaying progress on a particular task, use a Meter.
+
+### Accessibility
+
+A label is required when using Meter. Use one of these options:
+
+- Visible label using `MeterLabel`, with `htmlFor` set equal to the `id` of the Meter (preferred)
+- Visible label that's associated to the Meter with `aria-labelledby`
+- Label directly using `aria-label`
+
+## Examples
+
+### Default
+
+Use a Meter to communicate an amount of something within a range, like number of emails delivered. Use the `valueLabel` prop on the `MeterLabel` component to display the current value being represented by the Meter. Using multiple formats to represent the same value is not encouraged (e.g. "50 of 100 (50%)").
+
+<LivePreview scope={{Meter, MeterLabel, HelpText, useUID, Box}} language="jsx" noInline>
+  {defaultMeter}
+</LivePreview>
+
+### Min and max values
+
+Passing `minValue` and `maxValue` to Meter allow you to set a non 0-100 scale. Use `minLabel` and `maxLabel` to display minimum and maximum values below the Meter. If using a non 0-100 scale, displaying min and max labels is strongly encouraged.
+
+<LivePreview scope={{Meter, MeterLabel, HelpText, useUID, Box}} language="jsx" noInline>
+  {minMaxMeter}
+</LivePreview>
+
+### Hidden value label
+
+<LivePreview scope={{Meter, useUID, Box}} language="jsx" noInline>
+  {hiddenValueLabel}
+</LivePreview>
```

Comment:
question/suggestion: Is there a use case this is based on? Can you add words on what this is for, or remove it as an example on the doc site?